### DISCUSSION TITLE
knowledge-base/mesh-transport-platform/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#409] [Announcements]
# knowledge-base/mesh-transport-platform/

网址 网址 账户 密码 备注 http://123.57.153.47/ admin huami1701 测试环境管理端 http://123.57.153.47/protal/#/login 18888888888 123456 测试环境货主端

http://localhost:8080/knowledge-base/mesh-transport-platform/

**Comment by 2414163958** [comment]
很好
